src/test/kotlin/org/wfanet/measurement/loadtest/frontend/FrontendSimulatorImplTest.kt

```diff
@@ -24,9 +24,14 @@ import org.junit.rules.TemporaryFolder
 import org.junit.runner.RunWith
```

Comment:
Lets change the file name to FrontendSimulatorTest.kt . The Impl there was wrong naming